bedrock/firefox/firefox_details.py

```diff
@@ -310,8 +310,14 @@ class FirefoxAndroid(_ProductDetails):
     def platforms(self, channel='release'):
         platforms = self.platform_labels.copy()
 
+        # Android Gingerbread (2.3) is no longer supported as of Firefox 48
+        if int(self.latest_version(channel).split('.', 1)[0]) >= 48:
```

Comment:
Looks like you're doing this split twice. Would be better to set a variable like:

```python
major_version = int(self.latest_version(channel).split('.', 1)[0])
```